src/shipit_signoff/default.nix

```diff
@@ -22,8 +22,7 @@ let
       export APP_TESTING=${name}
 
       flake8 --exclude=nix_run_setup.py,migrations/,build/
-      #TODO: need to make tests work
-      #pytest tests/
+      pytest tests/
```

Comment:
I actually wonder if we shouldn't get rid of this file, now that tests are running. What do you think @garbas ?